allennlp/data/dataset_readers/brown_corpus.py

```diff
@@ -0,0 +1,79 @@
+from typing import Dict
+import logging
+
+from overrides import overrides
+import tqdm
+
+from allennlp.common import Params
+from allennlp.common.checks import ConfigurationError
+from allennlp.data.dataset import Dataset
+from allennlp.data.dataset_readers.dataset_reader import DatasetReader
+from allennlp.data.instance import Instance
+from allennlp.data.token_indexers.token_indexer import TokenIndexer
+from allennlp.data.fields import TextField, TagField
+from allennlp.data.token_indexers import SingleIdTokenIndexer
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+@DatasetReader.register("brown_corpus")
+class BrownCorpusDatasetReader(DatasetReader):
```

Comment:
Can we just use the `SequenceTaggingDatasetReader` here, instead of creating a new one?  If the only reason for making a new class is that the delimiter is different, we could just change the delimiter in the `SequenceTaggingReader`, or make it configurable.  The only reason the delimiter is `###` in the other reader is to disallow fewer tags / make collisions less likely.  But if the delimiter is configurable, this isn't really an issue.